apps/tb_meetings/src/tb_meetings_server.erl

```diff
@@ -0,0 +1,61 @@
+-module(tb_meetings_server).
+-behaviour(gen_server).
+-behaviour(deliverly_handler).
+-define(SERVER, ?MODULE).
+
+%% API Function Exports
+-export([start_link/0]).
+
+%% gen_server Function Exports
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2,
+         terminate/2, code_change/3]).
+
+%%% Deverly Handler callbacks.
+-export([authorize/2, handle_message/2, handle_client_message/2, client_disconnected/1]).
+
+%% API Function Definitions
+
+start_link() ->
+  gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
+
+%% deliverly_handler Function Definitions
+
+authorize(Client, Data) ->
+  case proplists:get_value(<<"meeting_id">>, Data) of
+    undefined ->
+      {error, 3401};
+    Id ->
+      case deliverly_utils:auth_from_config(tb_meetings, Client, Data) of
+        true -> gen_server:call(?SERVER, {authorize, binary_to_integer(Id), Client});
```

Comment:
Line 29 is too long:         true -> gen_server:call(?SERVER, {authorize, binary_to_integer(Id), Client});.